src/backend/base/langflow/components/models/embedding_model.py

```
@@ -1,16 +1,18 @@
-from typing import Any
+from typing import Any, List
```

Ruff (0.11.9)

`typing.List` is deprecated, use `list` instead

(UP035)